### ISSUE TITLE
Accessibility: Certain text on the home page has poor, inaccessible contrast

### ISSUE THREAD

**Issue by tai-jee** [zen-browser/www#898]
## "Made with ❤️ by the **Zen Team**" in the footer

First noticed this mentioned in [this Reddit thread](https://www.reddit.com/r/zen_browser/comments/1s021v5/made_with_by_the_who_my_eyes_cant_process_that/), and it doesn't look like anyone has made an actual GitHub issue regarding it as of yet.

<img width="391" height="206" alt="Image" src="https://github.com/user-attachments/assets/63463e4f-6341-4077-a272-19c7a785e819" />
<img width="340" height="220" alt="Image" src="https://github.com/user-attachments/assets/106ce44b-25f9-498b-a921-2be06edba9d6" />

As shown in the above image, the contrast ratio for the text here is 1.82:1, rather than WCAG's recommended minimum of 4.5:1. The second screenshot was taken inside the accessibility pane in the Inspect menu from within the Zen browser. This only seems to be the issue in dark mode, where the footer has a lighter background colour.

## "Welcome to a **calmer** internet" in the home page tagline

Also, upon checking the site myself after coming across that thread, I also noticed that the word "calmer" in the tagline on the home page (while in light mode) has contrast below WCAG standards (2.5, where I believe it should be at least 3 for larger text like this):

<img width="659" height="270" alt="Image" src="https://github.com/user-attachments/assets/04160ac3-0807-4c88-947d-323cb21d0369" /> 
<img width="347" height="213" alt="Image" src="https://github.com/user-attachments/assets/0adf7bda-71a0-444e-b1fa-f502ab64eb28" />

**Comment by puneetdixit200** [comment]
Hi @tai-jee 👋

I’d like to work on this issue.

I’ll look into the contrast problems in both the footer text ("Made with ❤️ by the Zen Team") and the homepage tagline, and update the styles to meet WCAG contrast guidelines in both dark and light modes.
